lib/x11test.pm

```diff
@@ -411,6 +411,33 @@ sub start_evolution {
     wait_still_screen(2);
 }
 
+sub start_evolution_from_backupfile {
+    my ($self, $mail_box) = @_;
+
+    # Cleanup past configs  and start Evolution.
+    x11_start_program("xterm -e \"killall -9 evolution; find ~ -name evolution | xargs rm -rf;\"", valid => 0);
+    x11_start_program('evolution', target_match => [qw(evolution-default-client-ask test-evolution-1 evolution-welcome-not_focused)]);
+
+    # Follow the wizard to setup mail account.
+    if (match_has_tag 'evolution-default-client-ask') {
+        assert_and_click "evolution-default-client-agree";
+        assert_screen "test-evolution-1";
+    }
+    elsif (match_has_tag "evolution-welcome-not_focused") {
```

Comment:
Does it work as well?
```suggestion
    else {
           assert_and_click "evolution-welcome-not_focused";
    }       
```

Reply:
When the Evolution app is started, the code is trying to match one of the three needles. So, it would be better to use 'elsif' to match the needle rather than 'else'.